packages/utils/src/route.js

```diff
@@ -41,29 +41,26 @@ export const flatRoutes = function flatRoutes (router, fileName = '', routes = [
   return routes
 }
 
-function cleanChildrenRoutes (routes, isChild = false, routeNameSplitter = '-', trailingSlash) {
-  let start = -1
+function cleanChildrenRoutes (routes, isChild = false, routeNameSplitter = '-', trailingSlash, parentRouteName) {
   const regExpIndex = new RegExp(`${routeNameSplitter}index$`)
+  const regExpParentRouteName = new RegExp(`^${parentRouteName}-`)
```

Comment:
Should use `routeNameSplitter` instead of hardcoded `-`.

Reply:
Thanks